google/resource_bigtable_instance.go

```diff
@@ -57,11 +61,9 @@ func resourceBigtableInstance() *schema.Resource {
 					},
 				},
 			},
-
 			"display_name": {
 				Type:     schema.TypeString,
 				Optional: true,
-				ForceNew: true,
```

Comment:
Since we haven't added the ability to update this field, we should keep it `ForceNew` right?

Reply:
Oh yeah, good catch this can be fixed in another PR